charts/stable/cloudflared/Chart.yaml

```
@@ -18,9 +18,9 @@ maintainers:
     url: https://truecharts.org
 name: cloudflared
 sources:
-  - https://github.com/truecharts/charts/tree/master/charts/incubator/cloudflared
+  - https://github.com/truecharts/charts/tree/master/charts/stable/cloudflared
 type: application
-version: 5.0.2
+version: 6.0.0
```

```suggestion
version: 5.0.3
```
move without changes is a chore, not a breaking change.